_posts/2015-06-26-ux-part-1.md

```diff
@@ -0,0 +1,69 @@
+
+##User-Centred Development
+
+There are a number of classical models of software development in the HCI world. OME probably comes closest to Hartson and Hix’s  Star Model, and although we have dabbled with Agile a bit, it is probably not the best model when there is no monolithic “client” for the software. However, this is not the place to delve into the details of user-centred design, but rather just pick up a couple of points, then show the process used in OME.
+
+The key to the UX approach is to make user-centred design pay in the long run by doing a number of good things: 
+
+- saving developer time and effort
+- avoiding mistakes and crossed wires
+- preventing redundancy
+- facilitating collaboration, and 
+- enhancing documentation.
+
+But actually the process does more! 
+
+Mental models are central to the way humans think, view and build understanding of the world around them and they are especially important in the use of computer software. As a rule mental models are incomplete, unstable, nebulous and easily confused, as frequently described by [Donald Norman](http://www.jnd.org). Because user-centred design requires mental models to be captured and put down on paper and talked about explicitly, it forces the conversion of implicit knowledge into explicit knowledge, stabailising the mental models and and clarifying ideas. 
+
+Particularly important for OME is the fact that mostly expert knowledge is “deep knowledge”, i.e. it is tacit, not consciously articulated and often relatively abstract. By getting experts to articulate their knowledge, it is made implicit and concrete. Work done by the late Joan Bliss showed that making abstract knowledge explicit actually changes it, and this is what we see when extracting expert knowledge and mental models from expert users.
+
+As expert knowledge is a special case of mental models, and with all the expert users and developers we have involved with OME, how it is handled is very important for us. Our key is to go back to the experts more than once to pick up the changes in their knowledge that happen after we have talked to them the first time. This is probably one of the main reasons why this particular facet of user-centred design really helps in expert system development. 
+
+Another phenomenon of user-centred design is that the process serves to prime the developer’s thought processes and get them thinking about the problem long before they narrow down and focus on the nuts and bolts of the actual implementation. Therefore before they start coding, the developers have done done a lot of pre-processing, firmed up their own mental models of the system, and filled in any gaps in their approach. As well as leading to an implementation that is likely to be close to an optimal solution, it can lead to real time-saving and efficient use of developer resources.
+
+**Part 2: The OME-UX Development Workflow.**
```

Comment:
I wonder whether this should have a 'coming next' or similar - I half-expected it to be a link